test/tbbmalloc/test_malloc_lib_unload.cpp

```diff
@@ -23,24 +23,28 @@
 const char *globalCallMsg = "A TBB allocator function call is resolved into wrong implementation.";
 
 #if _WIN32||_WIN64
+    #define TEST_EXPORT __declspec(dllexport)
+#else
+    #define TEST_EXPORT __TBB_EXPORT
+#endif
```

Comment:
Should we move this piece of code somewhere to common header so it can be used in all tests as `TEST_EXPORT`?

Reply:
In different tests, the values of the ``TEST_EXPORT`` macro are different (with and without ``__declspec(dllexport)``):

```
Searching 194 files for "__TBB_EXPORT" (case sensitive)

~/Devel/oneapi-src/oneTBB/test/tbb/test_dynamic_link.cpp:
   31  #if _WIN32 || _WIN64
   32  #define TEST_EXPORT
   33  #else
   34: #define TEST_EXPORT extern "C" __TBB_EXPORT
   35  #endif /* _WIN32 || _WIN64 */
   36  

~/Devel/oneapi-src/oneTBB/test/tbbmalloc/test_malloc_atexit.cpp:
   34  #if _WIN32||_WIN64
   35  extern __declspec(dllexport)
   36  #else
   37: __TBB_EXPORT
   38  #endif
   39  bool dll_isMallocOverloaded()

~/Devel/oneapi-src/oneTBB/test/tbbmalloc/test_malloc_lib_unload.cpp:
   25  #if _WIN32||_WIN64
   26      #define TEST_EXPORT __declspec(dllexport)
   27  #else
   28:     #define TEST_EXPORT __TBB_EXPORT
   29  #endif
   30  

~/Devel/oneapi-src/oneTBB/test/tbbmalloc/test_malloc_used_by_lib.cpp:
   27  #if _WIN32||_WIN64
   28      extern __declspec(dllexport) void callDll();
   29  #else
   30:     extern __TBB_EXPORT void callDll();
   31  #endif
   32  }

4 matches across 4 files
```

I think in this PR such a generalization does not make sense.